spec/classes/server_spec.rb

```diff
@@ -252,7 +252,7 @@
             }
           end
 
-          it { is_expected.to contain_file('/root/.mongorc.js').with_ensure('absent') }
+          it { is_expected.to contain_file('/root/.mongorc.js') }
```

Comment:
Similar here: verify that the file exists but without the actual credentials.

Reply:
Valid point. Line updated.